builtin/providers/google/resource_storage_bucket_object.go

```diff
@@ -151,14 +217,6 @@ func resourceStorageBucketObjectDelete(d *schema.ResourceData, meta interface{})
 	err := DeleteCall.Do()
 
 	if err != nil {
-		if gerr, ok := err.(*googleapi.Error); ok && gerr.Code == 404 {
-			log.Printf("[WARN] Removing Bucket Object %q because it's gone", name)
-			// The resource doesn't exist anymore
-			d.SetId("")
-
-			return nil
-		}
-
```

Comment:
I...don't actually know where this is from, i think it was a last minute merge issue that I didn't catch. I'll fix this.